bjoern/_bjoernmodule.c

```diff
@@ -10,9 +11,22 @@ run(PyObject* self, PyObject* args)
 
   PyObject* socket;
 
-  if(!PyArg_ParseTuple(args, "OO:server_run", &socket, &info.wsgi_app)) {
+#ifdef WANT_STATSD
+  StatsdInfo statsd_info;
```

Comment:
Let's make this local variables with `statsd_` prefix, not a struct, since it's only used temporarily in this routine.